stratum/hal/config/config_validator_main.cc

```diff
@@ -27,12 +27,14 @@ TEST(ConfigValidator, TestChassisConfig) {
   EXPECT_OK(ReadProtoFromTextFile(filename, &chassis_config));
 }
 
+#ifndef SIM_TARGET
 TEST(ConfigValidator, TestPhalConfig) {
   std::string filename = absl::StrFormat(
       "stratum/hal/config/%s/phal_config.pb.txt", STRINGIFY(PLATFORM));
   PhalInitConfig phal_config;
   EXPECT_OK(ReadProtoFromTextFile(filename, &phal_config));
 }
```

Comment:
This test should be optional anyway. A phal init config is only needed when the platform structure is not discoverable (i.e. broken or no ONLP).